RecoLocalTracker/SiPixelRecHits/plugins/SiPixelRecHitFromCUDA.cc

```diff
@@ -85,7 +85,8 @@ void SiPixelRecHitFromCUDA::acquire(edm::Event const& iEvent,
 void SiPixelRecHitFromCUDA::produce(edm::Event& iEvent, edm::EventSetup const& es) {
   // allocate a buffer for the indices of the clusters
   auto hmsp = std::make_unique<uint32_t[]>(gpuClustering::maxNumModules + 1);
-  std::copy(hitsModuleStart_.get(), hitsModuleStart_.get() + gpuClustering::maxNumModules + 1, hmsp.get());
+  if (hitsModuleStart_)
```

Comment:
where is `hitsModuleStart_` (re)set?
it looks like for `nHits_ == 0` in the current event, but a previous event had it non-zero, the `hitsModuleStart_` will be set and point to the previous event.
Wouldn't it be more direct to test for `nHits_ == 0` here?

Reply:
ping (this bugfix seemed urgent)

Reply:
@czangela